simphony/cuds/material.py

```diff
@@ -0,0 +1,31 @@
+import uuid
+
+from simphony.core.data_container import DataContainer
+
+
+class Material(object):
+    """A class that represents a material.
+
+    Class describes a material and its data.
+
+    Attributes
+    ----------
+    description : str
+        textual description of the material
+    uid : uuid.UUID
```

Comment:
A minor thing, but maybe the attributes list here should be in the same order as in the uml image (material.txt).